iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.export;
+
+import com.powsybl.iidm.network.TopologyLevel;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class ExportOptions {
```

Comment:
To avoid cast later, it could be interesting to use the CRTP pattern:
```
public class ExportOptions<E extends ExportOptions<E>> { ... }
```

All the methods which return an instance of ExportOptions in this class should return an instance of E to avoid casting:
```
    public E setWithBranchSV(boolean withBranchSV) {
        this.withBranchSV = withBranchSV;
        return (E) this;
    }
```

Reply:
Another idea is to handle the missing parameter (indent) in your JSON exporter:
```
ObjectMapper objectMapper = new ObjectMapper();
objectMapper.enable(SerializationConfig.Feature.INDENT_OUTPUT);
```
and to remove the XMLExportOptions specialization

Reply:
Regarding this issue, I kept only the ExportOptions class and handled indentation as an option for the JSON exporter as well.